kong/plugins/prometheus/prometheus.lua

```diff
@@ -179,30 +179,40 @@ local function full_metric_name(name, label_names, label_values)
     return name
   end
 
-  local label_parts = tb_new(#label_names, 0)
-  local label_value
+  -- format "name{k1=v1,k2=v2}"
+  local buf = buffer.new()
```

Comment:
Additional thing we could do, but not neccessary, is to record previous run buffer size and use that as a `hint` to `new`.

Reply:
Good idea, I will think about it, thank you.